test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1

```diff
@@ -395,15 +379,13 @@ try
             finally
             {
                 if ($restoreEAPreference -ne $null) { $ErrorActionPreference = $restoreEAPreference }
-                Invoke-LanguageModeTestingSupportCmdlet -RevertLockdownMode -EnableFullLanguageMode
+                Invoke-LanguageModeTestingSupportCmdlet -EnableFullLanguageMode
             }
 
-            $exception.FullyQualifiedErrorId | Should Match "MethodInvocationNotSupportedInConstrainedLanguage"
+            $exception.FullyQualifiedErrorId | Should Match "CannotSetBreakpointInconsistentLanguageMode"
```

Comment:
Can this be changed to use ShouldBeErrorId pattern?  Same for all the exception validation below.

Reply:
No